ttnn/cpp/ttnn/operations/eltwise/binary_ng/binary_ng.cpp

```diff
@@ -0,0 +1,55 @@
+
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "binary_ng.hpp"
+
+namespace ttnn::operations::binary_ng {
+
+template <BinaryOpType binary_op_type>
+Tensor BinaryNg<binary_op_type>::invoke(
+    uint8_t queue_id,
+    const Tensor &input_tensor_a,
+    const Tensor &input_tensor_b,
+    const std::optional<const DataType> &output_dtype,
+    const std::optional<MemoryConfig> &memory_config,
+    std::optional<Tensor> optional_output_tensor) {
+    return ttnn::prim::binary_ng(
```

Comment:
What are your thoughts on the need for a secondary binding outside of ttnn::prim?
It looks totally useless now as operation requires no conditional repeats